### PR TITLE
Little Improvements in de_DE.lang

### PR DESCRIPTION
**Pull request by Handfeger**
I made a Localization for German too but didn't come first. Here is now a merged Version between @Galaxy2Alex Version and mine, with the best of both translations.

I also restored the look of `en_US.lang` so the line numbers are matching in both Translations.

### CODE REVIEW

**Comment by Vswe** [comment]
Accepting the other pull request instead as agreed upon

**Comment by Vswe** [comment]
Ehm, might have been a bit confused. So many people want to help with the german version. Have you been in contact with @Galaxy2Alex ?

**Comment by Handfeger** [comment]
No not jet, I took his version and put in a few improvements. ae0e0a0 is the merged version with mostly his changes.

Maybe @Galaxy2Alex can look over my changes and confirm them.

**Comment by Galaxy2Alex** [comment]
Currently improving, thanks.
I would propose a Skype group, just add me via the E-Mail adress "galaxy2alex@gmail.com"

**Comment by Handfeger** [comment]
Added you to Skype

**Comment by Vswe** [comment]
If the 3 of you (@SHoCKYCooKIE has been involved in the current version) get together as a team (like on Skype or whatever) that would be nice. That would be the official German translation team and I would automatically approve your pull requests as I receive them. Does that sound good to you?

File: de_DE.lang [hunk 1]
```diff
@@ -200,9 +210,8 @@ item.SC2:extracting_chests.name=Ausfahrende Kisten
 item.SC2:torch_placer.name=Fackelsetzer
 item.SC2:basic_drill.name=Grundbohrer
 item.SC2:galgadorian_drill.name=Galgadorbohrer
-item.SC2:railer.name=Aufgleisgerät
-item.SC2:large_railer.name=Großes Aufgleisgerät
-item.SC2:side_tanks.name=Side Tanksroßes Aufgleisgerät
```
Comment (Handfeger) [inline de_DE.lang:205]: Moved to line 267

**Comment by Handfeger** [comment]
Sounds good maybe we should bundle our Translation into @Galaxy2Alex Repo before sending pull requests. Merging would be much easier.

**Comment by Galaxy2Alex** [comment]
...and there's Skype being Skype...
Yes, we're in a Skype conversation now.

**Comment by Vswe** [comment]
Ok, nice. I'll close this pull request again (doesn't block us from commenting in it though).

**Pull request closed without merging**